migrations/V58__Update_Plan_Ops_Eval_and_Learning_To_User_Account_Table.sql

```diff
@@ -0,0 +1,80 @@
+RENAME COLUMN ready_for_clearance_by TO ready_for_clearance_by_old;
+
+ALTER TABLE plan_ops_eval_and_learning
+RENAME COLUMN ready_for_review_by TO ready_for_review_by_old;
+
+ALTER TABLE plan_ops_eval_and_learning
+RENAME COLUMN created_by TO created_by_old;
+
+
+ALTER TABLE plan_ops_eval_and_learning
+RENAME COLUMN modified_by TO modified_by_old;
+
+
+/* ADD Correct Column */
+ALTER TABLE plan_ops_eval_and_learning
+ADD COLUMN ready_for_clearance_by UUID REFERENCES public.user_account (id) MATCH SIMPLE,
+ADD COLUMN ready_for_review_by UUID REFERENCES public.user_account (id) MATCH SIMPLE,
+ADD COLUMN created_by UUID REFERENCES public.user_account (id) MATCH SIMPLE,
+ADD COLUMN modified_by UUID REFERENCES public.user_account (id) MATCH SIMPLE;
+
+ALTER TABLE plan_ops_eval_and_learning
+DISABLE TRIGGER audit_trigger;
+
+/* Perform the data migration */
+WITH userAccount AS (
+    SELECT
+        plan_ops_eval_and_learning.id AS primaryID,
+        user_account_ready_for_clearance.id AS ready_for_clearance_by,
+        user_account_ready_for_review.id AS ready_for_review_by,
+        user_account_created.id AS created_by,
+        user_account_modified.id AS modified_by
+    FROM plan_ops_eval_and_learning
+    LEFT JOIN user_account AS user_account_ready_for_clearance ON plan_ops_eval_and_learning.ready_for_clearance_by_old = user_account_ready_for_clearance.username
+    LEFT JOIN user_account AS user_account_ready_for_review ON plan_ops_eval_and_learning.ready_for_review_by_old = user_account_ready_for_review.username
+    LEFT JOIN user_account AS user_account_created ON plan_ops_eval_and_learning.created_by_old = user_account_created.username
+    LEFT JOIN user_account AS user_account_modified ON plan_ops_eval_and_learning.modified_by_old = user_account_modified.username
+)
+--
+
+UPDATE plan_ops_eval_and_learning
+SET
+    created_by = userAccount.created_by,
+    modified_by = userAccount.modified_by,
+    ready_for_clearance_by = userAccount.ready_for_clearance_by,
+    ready_for_review_by = userAccount.ready_for_review_by
+FROM userAccount
+WHERE userAccount.primaryID = plan_ops_eval_and_learning.id;
+
+
+/*remove the old columns */
+ALTER TABLE plan_ops_eval_and_learning
+DROP COLUMN ready_for_clearance_by_old,
+DROP COLUMN ready_for_review_by_old,
+DROP COLUMN created_by_old,
+DROP COLUMN modified_by_old;
+
+/*add constraints */
+ALTER TABLE plan_ops_eval_and_learning
+ALTER COLUMN created_by SET NOT NULL;
+
+
+/* update audit config */
+UPDATE audit.table_config
+SET uses_user_id = TRUE,
+    modified_by = '00000001-0001-0001-0001-000000000001', --System Account
+    modified_dts = current_timestamp
+WHERE name = 'plan_ops_eval_and_learning';
+
+/* turn on audit trigger */
+
+ALTER TABLE plan_ops_eval_and_learning
+ENABLE TRIGGER audit_trigger;
+
+/* disable need trigger as it is currently non-functional until needs are migrated */
+
+ALTER TABLE plan_ops_eval_and_learning
+DISABLE TRIGGER operational_need_trigger;
```

Comment:
I think (though functionally equivalent) it makes sense to make this part of the migration V59.

The rest of this migration (as the name implies) is migrating the Ops Eval Learning section to the user table, but this part is responsible for something else entirely (disabling functionality while we write future migrations)

To ensure that this block is easy to find (and write the corresponding `enable trigger` migration later, can you break this out into a new migration?

Reply:
Thanks Clay! Good call. I was disabling the trigger for each table in each user table migration, but I think I'm going to include disabling the trigger on each table in this new migration so it's all consolidated.